reporting/impl/src/main/java/org/jboss/windup/reporting/freemarker/problemsummary/GetProblemSummariesMethod.java

```diff
@@ -99,6 +112,10 @@ public int compare(String severity1, String severity2)
         ClassificationService classificationService = new ClassificationService(context);
         for (ClassificationModel classification : classificationService.findAll())
         {
+            Set<String> tags = classification.getTags();
+            if (!TagUtil.isTagsMatch(tags, includeTags, excludeTags, false))
```

Comment:
maybe it should be IssueFilter that will just happen to filter issues with the given tags out.